### PR TITLE
DEVDOCS-3787-add-API-option

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-3787](https://jira.bigcommerce.com/browse/DEVDOCS-3787)

## What changed?
Update the following dev docs by adding that installing custom checkouts and custom order confirmation page can now be done via an [API](https://developer.bigcommerce.com/api-reference/b3A6NDkxOTE4NDQ-update-checkout-settings)

### CODE REVIEW

File: docs/api-docs/cart-and-checkout/custom-order-confirmation.md [hunk 1]
```diff
@@ -61,6 +61,11 @@ You can upload a custom order confirmation page to your store's server using Web
 
 ## Installing a custom order confirmation page  
 
+There are two ways to install a custom checkout on a store.
+* Installing using the control panel
+* Installing using the Checkout v3 API
+
+### Installing using the control panel
```
Comment (slsriehl) [inline docs/api-docs/cart-and-checkout/custom-order-confirmation.md:68]:
```suggestion
### Install using the control panel
```
style guide

File: docs/api-docs/cart-and-checkout/custom-order-confirmation.md [hunk 2]
```diff
@@ -84,6 +89,10 @@ To install a custom order confirmation page on a store, follow these steps:
 3. Click the **Save** button at the bottom of the page.
 4. Navigate to your live storefront to view your new custom order confirmation page.
 
+### Installing using the Checkout v3 API
```
Comment (slsriehl) [inline docs/api-docs/cart-and-checkout/custom-order-confirmation.md:92]:
```suggestion
### Install using the Checkout v3 API
```
style guide

Comment (slsriehl) [reply docs/api-docs/cart-and-checkout/custom-order-confirmation.md:92]: also, in many of the docs I've been working on, I see version before API name -- for example, "V3 Checkout API". With the V capitalized. Is either of these proscribed by the style guide, @traci707 / @markcmurphy

File: docs/api-docs/cart-and-checkout/custom-order-confirmation.md [hunk 3]
```diff
@@ -61,6 +61,11 @@ You can upload a custom order confirmation page to your store's server using Web
 
 ## Installing a custom order confirmation page  
 
+There are two ways to install a custom checkout on a store.
```
Comment (slsriehl) [inline docs/api-docs/cart-and-checkout/custom-order-confirmation.md:64]:
```suggestion
You can install a custom checkout on a store in the following two ways:
```
this revision ^^ is one solution to make the following bullet points less wordy. you can also make them anchor links to the sections below.

**Comment by slsriehl** [comment]
I added some notes. Parallel notes for the second document.

File: docs/api-docs/cart-and-checkout/custom-order-confirmation.md (hunk 2 above)
Comment (traci707) [reply docs/api-docs/cart-and-checkout/custom-order-confirmation.md]: @slsriehl I did not find information in the style guide but I like your suggestion and think it looks good.

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**